metrics/stats.go

```diff
@@ -69,20 +74,46 @@ var lastPauseNs uint64
 var lastNumGc uint32
 
 var connections int64
+var connectionsSSE int64
+var connectionsGRPC int64
 
 func IncConnection() {
 	atomic.AddInt64(&connections, 1)
 }
 
+func IncConnectionSSE() {
+	atomic.AddInt64(&connectionsSSE, 1)
+}
+
+func IncConnectionGRPC() {
+	atomic.AddInt64(&connectionsSSE, 1)
+}
+
 func DecConnection() {
 	atomic.AddInt64(&connections, -1)
 }
 
+func DecConnectionSSE() {
+	atomic.AddInt64(&connectionsSSE, -1)
+}
+
+func DecConnectionGRPC() {
+	atomic.AddInt64(&connectionsGRPC, -1)
+}
+
 func GetConnection() int64 {
 	return atomic.LoadInt64(&connections)
 }
 
-func GetStats() *Stats {
+func GetConnectionSSE() int64 {
+	return atomic.LoadInt64(&connectionsSSE)
+}
+
+func GetConnectionGRPC() int64 {
```

Comment:
__[golint]__
* [golint] exported function GetConnectionGRPC should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->